state/state.go

```diff
@@ -14,6 +14,7 @@ import (
 type Client interface {
 	SetKey(ctx context.Context, key string, value string) error
 	GetKey(ctx context.Context, key string) (string, error)
+	Flush(ctx context.Context) error
```

Comment:
What's flush for?

Reply:
It is not flushing the content every `Set` otherwise data is not written back. It is already implemented on the `state` implementation but just wasn't exposed